logger.py

```diff
@@ -1,26 +1,26 @@
 import logging
 import os
 
-class Logger:
-    def __init__(self):
-        # Prod mode
-        if not os.getenv("DEV_MODE"):
-            import google.cloud.logging
 
-            client = google.cloud.logging.Client()
-            client.setup_logging()
+def init_logger():
+    # Prod mode
+    if not os.getenv("DEV_MODE"):
+        import google.cloud.logging
```

Comment:
wait why import it here?

Reply:
its okay to import it at the top unless there is like a package conflict or import conflict imo